bots/image-refresh

```diff
@@ -97,7 +97,8 @@ STORES = {
     "rhel-7-6": REDHAT_STORE,
     "rhel-x": REDHAT_STORE,
     "rhel-atomic": REDHAT_STORE,
-    "windows-8": REDHAT_STORE
+    "windows-8": REDHAT_STORE,
+    "windows-10": REDHAT_STORE
```

Comment:
This should be in the previous commit that adds the image. And please add a trailing comma while you are at it :)

Reply:
will do :)